pkgs/top-level/all-packages.nix

```diff
@@ -5973,6 +5973,10 @@ with pkgs;
 
   html-proofer = callPackage ../tools/misc/html-proofer { };
 
+  htmlq = callPackage ../development/tools/htmlq {
+    inherit (pkgs.darwin.apple_sdk.frameworks) Security;
```

Comment:
```suggestion
    inherit (darwin.apple_sdk.frameworks) Security;
```

I think `pkgs` is not required.

Reply:
Oh I just saw this comment. Looks like there are several occurrences of this in `all-packages.nix` so a file-wide change might be in order.

Reply:
Ah, doesn't look like there is much of a convention.  

>Looks like there are several occurrences of this in all-packages.nix so a file-wide change might be in order.

Good call.